### PR TITLE
Fix emc2101_enums import

### PR DESCRIPTION
**Pull request by caternuson**
For #31.

Tested on QT PY RP2040:
```python
Adafruit CircuitPython 8.2.2 on 2023-07-31; Adafruit QT Py RP2040 with rp2040
>>> import board
>>> from adafruit_emc2101 import EMC2101
>>> i2c = board.STEMMA_I2C()
>>> emc = EMC2101(i2c)
>>> emc.spinup_drive
3
>>> emc.spinup_drive = 0
>>> emc.spinup_drive
0
>>> 
```

### CODE REVIEW

**Comment by vlycop** [comment]
Hi ! OP here
What about L413 and L478 ? Should they be fixed to ?

Also i am very surprise that any other part of this library don't need this `.`

Not patching the other line cause spinup_time to not work
```python
(venv) pi@prusabox:~/test_emc2101 $ vi /home/pi/test_emc2101/venv/lib/python3.7/site-packages/adafruit_emc2101/__init__.py 
(venv) pi@prusabox:~/test_emc2101 $ rm -r /home/pi/test_emc2101/venv/lib/python3.7/site-packages/adafruit_emc2101/__pycache__
(venv) pi@prusabox:~/test_emc2101 $ python
Python 3.7.3 (default, Jun 29 2023, 18:03:57) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import board
>>> from adafruit_emc2101.emc2101_lut import EMC2101_LUT as EMC2101
>>> i2c = board.I2C()
>>> emc = EMC2101(i2c)
>>> emc.spinup_drive
1
>>> emc.spinup_drive = 3
>>> emc.spinup_drive
3
>>> emc.spinup_time
4
>>> emc.spinup_time = 7
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/pi/test_emc2101/venv/lib/python3.7/site-packages/adafruit_emc2101/__init__.py", line 413, in spinup_time
    from emc2101_enums import SpinupTime
ModuleNotFoundError: No module named 'emc2101_enums'
```
while doing so fix the issue
```python
(venv) pi@prusabox:~/test_emc2101 $ vi /home/pi/test_emc2101/venv/lib/python3.7/site-packages/adafruit_emc2101/__init__.py 
(venv) pi@prusabox:~/test_emc2101 $ rm -r /home/pi/test_emc2101/venv/lib/python3.7/site-packages/adafruit_emc2101/__pycache__
(venv) pi@prusabox:~/test_emc2101 $ python
Python 3.7.3 (default, Jun 29 2023, 18:03:57) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import board
>>> from adafruit_emc2101.emc2101_lut import EMC2101_LUT as EMC2101
>>> i2c = board.I2C()
>>> emc = EMC2101(i2c)
>>> emc.spinup_drive
3
>>> emc.spinup_drive = 1
>>> emc.spinup_drive
1
>>> emc.spinup_time
7
>>> emc.spinup_time = 4
>>> emc.spinup_time
4
>>>
```

**Comment by caternuson** [comment]
Related forum post:
https://forums.adafruit.com/viewtopic.php?t=204056

Good catch. Those other imports should also be updated. Pushed those changes.

File: adafruit_emc2101/__init__.py [hunk 1]
```diff
@@ -410,7 +410,7 @@ def spinup_time(self, spin_time):
         # Not importing at top level so the SpinupTime is not loaded
         # unless it is required, and thus 1KB bytecode can be avoided.
         # pylint: disable=import-outside-toplevel
-        from emc2101_enums import SpinupTime
+        from .emc2101_enums import SpinupTime
```
Comment (vlycop) [inline adafruit_emc2101/__init__.py]:
```suggestion
        from adafruit_emc2101.emc2101_enums import SpinupTime
```
Is this the same or better ? both work, and i'm not a dev ><

File: adafruit_emc2101/__init__.py [hunk 2]
```diff
@@ -443,7 +443,7 @@ def spinup_drive(self, spin_drive):
         # Not importing at top level so the SpinupDrive is not loaded
         # unless it is required, and thus 1KB bytecode can be avoided.
         # pylint: disable=import-outside-toplevel
-        from emc2101_enums import SpinupDrive
+        from .emc2101_enums import SpinupDrive
```
Comment (vlycop) [inline adafruit_emc2101/__init__.py]:
```suggestion
        from adafruit_emc2101.emc2101_enums import SpinupDrive
```

File: adafruit_emc2101/__init__.py [hunk 3]
```diff
@@ -473,7 +473,7 @@ def conversion_rate(self, rate):
         # Not importing at top level so the ConversionRate is not loaded
         # unless it is required, and thus 1KB bytecode can be avoided.
         # pylint: disable=import-outside-toplevel
-        from emc2101_enums import ConversionRate
+        from .emc2101_enums import ConversionRate
```
Comment (vlycop) [inline adafruit_emc2101/__init__.py]:
```suggestion
        from adafruit_emc2101.emc2101_enums import ConversionRate
```

**Review by vlycop — COMMENTED** [review]
I am very confused by that random `.` so i started looking around...

I have no idea if this is better or not, but bot work so any of those version could be merged into a pip release :)

**Comment by caternuson** [comment]
TBH - This is one of the python-isms I'm never sure about myself. I actually like your approach better though.

**Review by FoamyGuy — APPROVED** [review]
This looks good to me. Thanks for the fix @caternuson  and reporting the issue @vlycop 

I pushed one commit to tweak to use the full name for the import. I am in agreement with both of you about preferring that one, though I also do not know if there is a technical difference.

**Comment by caternuson** [comment]
@FoamyGuy thanks for updating and closing this one up!

**Pull request merged**